src/database.ts

```diff
@@ -1034,20 +1197,20 @@ class Database extends ServiceObject {
    * @param {object} config Request config
    * @returns {Stream}
    */
-  makePooledStreamingRequest_(config) {
+  makePooledStreamingRequest_(config: RequestConfig): Readable {
     const self = this;
     const pool = this.pool_;
-    let requestStream;
-    let session;
+    let requestStream: ClientDuplexStream<RequestConfig, Readable>;
```

Comment:
Pretty sure this is incorrect, the first generic type should be the grpc message being sent and the second should be the grpc message received. Making this method a generic might be more trouble than it is worth (unless you disagree?) so I might just use a plain `Duplex` here instead.

Reply:
SGTM